entrypoint.sh

```diff
@@ -46,5 +46,13 @@ echo "=> Updating Youtube-dl..."
 $YTDL_BIN -U
 echo "=> Updated youtube-dl with exit code $?"
 
+echo "=> Updating Sinusbot"
+cd /sinusbot
+wget https://www.sinusbot.com/dl/sinusbot.current.tar.bz2
+tar -xjvf sinusbot.current.tar.bz2
+cp plugin/libsoundbot_plugin.so TeamSpeak3-Client-linux_amd64/plugins/
+chown -R sinusbot /sinusbot
+echo "=> Done"
+
```

Comment:
This should be removed. With this added, you cannot run the image in an environment without internet.
Let me know if you are okay with that.

Reply:
Is it about the copy or the update part in general?

Its good to use the latest version of the bot because when its outdated you won't receive any support when something is not working. The problem is that you use the docker build because you dont want to spend time on keeping the bot updated, in general you wont think about it. So insted of manually updating the bot or updating the docker build and start again, it could be updated on every start. I thought it was a good idea because who needs a music bot and teamspeak server on a local network without access to the internet.

But I see your point. It may be better to ping a website to see if the connection is working and then trying to update instead of doing it anyway.

Reply:
@Horstexplorer The image is rebuild from time to time, so it gets the latest version of Sinusbot "automatically". The user simply pulls the image to get the latest version.
You should not download something in the container on start it is not good to do.

Reply:
I dont have a real experience in working with dockers so i dont know. If i download the image and create a container from it and now i would update the image i need to create a new container. Not sure if im right with it
So i would have to newly configure my sinusbot every time i want to update it to a new version.
Or does it keep my changes?
Would be very annoying for the user to reupload the music

Reply:
No, that is what a volume is for the container. The volume is mounted to the host and persists the music and data of Sinusbot. See https://github.com/galexrt/docker-sinusbot#mount-host-directory.
You just re-pull the image and restart the container then.

You wrote that you don't have much experience with Docker, I recommend these Docker guides to you:
* https://docs.docker.com/get-started/
* https://opensource.com/business/14/7/guide-docker
* https://docker-curriculum.com/